Formula/openjfx.rb

```diff
@@ -0,0 +1,89 @@
+# For `pour_bottle? do` block in main formula body below:
+class XcodeCLTRequirement < Requirement
+  fatal true
+
+  satisfy(:build_env => false) do
+    MacOS::CLT.installed?
+  end
+
+  def message
+    <<~EOS
+      This software requires the Xcode CLT in order to build.
+    EOS
+  end
+end
+
+class Openjfx < Formula
+  desc "Open-source, next-generation Java client application platform."
+  homepage "https://openjfx.io/"
+  url "http://hg.openjdk.java.net/openjfx/jfx-dev/rt", :revision => "22c69e868654", :using => :hg
```

Comment:
Can't use a checkout and tarballs at the same time, mirrors need to have identical content.

https://github.com/javafxports/openjdk-jfx
Did you try the GitHub mirror?

Reply:
@commitay:  

> Can't use a checkout and tarballs at the same time, mirrors need to have identical content.
>
> ⋮

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Thanks, that answers [my Discourse question](https://discourse.brew.sh/t/different-checksum-for-formula-package-download-mirror/3174).  Could you also leave this information there, please, or let me quote your response here there?  In any case, which of the various download sources would you prefer?  

> ⋮
>
> https://github.com/javafxports/openjdk-jfx
> Did you try the GitHub mirror?

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Yes, that's the second download mirror URL.

Reply:
@commitay:  

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Sorry if it wasn't clear, but that was only with respect to that Discourse thread.  I still need to know:  

> …which of the various download sources would you prefer?

Reply:
Depends really, how is it versioned? The current revision doesn't look like it corresponds to a tag?

Reply:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Well, I was thinking that the version number would need to match the '11+28' of the current JDK release, but I checked the build distributed on the OpenJFX as part of its official standalone JavaFX SDK package…and, according to that ZIP artifact's contained '`lib/javafx.properties`' file, said build's version is '11+26.'  The GitHub repository mirror has a tag for that, so I'll just use one of the archives associated with it.